src/features/create-dao/steps/DetailsForm.module.scss

```diff
@@ -0,0 +1,28 @@
+.Row {
+	display: flex;
+}
+
+.Column {
+	display: flex;
+	flex-direction: column;
+}
+
+.MediaInput {
+	height: 12.5rem;
+	width: 12.5rem;
+	margin-right: 1.5rem;
+	border-radius: 0.5rem !important;
```

Comment:
You can import variables from zUI for things like borders. something like..

```
@import '../../../../node_modules/@zero-tech/zui/styles/_borders.scss';
```

There should be a few examples within this and other zApps 

```
border-radius: calc($radius-primary / 2);
```